docs/functionalabstractions/concretetypes/ord.md

```diff
@@ -3,4 +3,190 @@ id: ord
 title: "Ord"
 ---
 
-Coming soon.
+`Ord[A]` describes a total ordering on values of type `A`.
+
+Its signature is:
+
+```scala mdoc
+trait Equal[-A] {
+  def equal(left: A, right: A): Boolean
+}
+
+trait Ord[-A] {
+  def compare(left: A, right: A): Ordering
+  final def equal(left: A, right: A): Boolean =
+    compare(left, right) == Ordering.Equals
+}
+
+sealed trait Ordering
+object Ordering {
+  case object LessThan extends Ordering
+  case object GreaterThan extends Ordering
+  case object Equals extends Ordering
+}
+```
+
+That is, we can compare any two values of type `A` and either the `left` value is less than the `right`, the `left` value is greater than the `right`, or the two values are equal.
+
+Furthermore, the results of these comparisons must satisfy the properties that we expect of a total ordering. For example, if `a` is greater than `b` and `b` is greater than `c` then `a` must be greater than `c` and if `a` is greater than `b` then `b` must be less than `a`.
```

Comment:
you are really speaking about **transitivity** of the relation here not total ordering. Although the ideas are related given the method `compare(left: A, right; A): Ordering` which is a total function rather than a partial function. But the way you have it written looks a little imprecise.

Reply:
One of the properties a total ordering has to have is it is a transitive relationship. You're right I'm trying to talk about the properties a total ordering has to have without actually listing out all of them.